src/components/WarningBanner/index.tsx

```diff
@@ -4,13 +4,15 @@ import { ReactComponent as InformationCircleIcon } from '@/assets/img/hero-icons
 import { KUSAMA, POLKADOT } from '@/utils/constants/relay-chain-names';
 import { BORDER_CLASSES } from '@/utils/constants/styles';
 
+type WarningLevel = 'alert' | 'info';
+
 interface CustomProps {
-  message: string;
+  warningLevel: WarningLevel;
```

Comment:
NIT: Im using `severity` in the score component. Would be cool to reuse it here.

Reply:
Will make that change - this component will be deprecated, but makes sense to use the same term even in the short term